jans-auth-server/client/src/test/java/io/jans/as/client/client/Asserter.java

```diff
@@ -51,15 +51,21 @@ public static void assertOk(RegisterResponse response) {
         assertEquals(response.getStatus(), 201, "Unexpected response code: " + response.getEntity());
         assertNotNull(response.getClientId());
         assertNotNull(response.getClientSecret());
-        assertNotNull(response.getClientIdIssuedAt());
```

Comment:
Why this assert is removed? I guess issued at is always present when client is created, no?

Reply:
assertNotNull(response.getClientIdIssuedAt());

was not in most of the asserts that I have replaced, I preferred do not include this assert to avoid problems in current test cases.
I preserved this assert in cases that it was present.
I don't know if this assert was omitted due to specific reasons, or it was forgotten.
who I have to ask to confirm that issuedAt is always present when client is created?

Reply:
You can ask me. Please add it.

Reply:
it was added.